### ISSUE TITLE
Instructions incorrect. 

### ISSUE THREAD

**Issue by AlbertoMQ** [birehan/Optimized-Contract-QA-RAG-System-Enhancements#10]
There is no npm start script or backend/tests. It feels like the Readme was written by AI

**Comment by birehan** [comment]
You’re right; the README needs updates. I’ll correct it with the necessary details. Thanks for pointing that out!

**Comment by AlbertoMQ** [comment]
I appreciate it!
